### ISSUE TITLE
CMAKE ERROR  error: <type 'exceptions.ValueError'>: common dialect not listed!

### ISSUE THREAD

**Issue by yajingwang94** [mavlink/mavros#659]
Hi, I git the package mavlink/mavros and mavlink/mavros to support the catkin_make process of the [rotors_simulator](https://github.com/ethz-asl/rotors_simulator) package, but it reports the following error which I have never met:

![Uploading Screenshot from 2017-02-20 21:34:30.png…]()
`<file:0 (/home/u23/WorkSpace/04.fixed_wing_sim/build/mavros/libmavconn/catkin_generated/mavlink.context.py)>:1: error: <type 'exceptions.ValueError'>: common dialect not listed!
Traceback (most recent call last):
  File "/usr/bin/empy", line 3276, in <module>
    if __name__ == '__main__': main()
  File "/usr/bin/empy", line 3274, in main
    invoke(sys.argv[1:])
  File "/usr/bin/empy", line 3238, in invoke
    interpreter.wrap(command, (target, name))
  File "/usr/bin/empy", line 2262, in wrap
    self.fail(e)
  File "/usr/bin/empy", line 2253, in wrap
    callable(*args)
  File "/usr/bin/empy", line 2358, in string
    self.safe(scanner, True, locals)
  File "/usr/bin/empy", line 2368, in safe
    self.parse(scanner, locals)
  File "/usr/bin/empy", line 2388, in parse
    token.run(self, locals)
  File "/usr/bin/empy", line 1403, in run
    interpreter.execute(self.code, locals)
  File "/usr/bin/empy", line 2565, in execute
    exec(statements, self.globals)
  File "<string>", line 1, in <module>
  File "<string>", line 16, in <module>
ValueError: common dialect not listed!
CMake Error at /opt/ros/jade/share/catkin/cmake/safe_execute_process.cmake:11 (message):
  
  execute_process(/home/u23/WorkSpace/04.fixed_wing_sim/build/catkin_generated/env_cached.sh
  "/usr/bin/python" "/usr/bin/empy" "--raw-errors" "-F"
  "/home/u23/WorkSpace/04.fixed_wing_sim/build/mavros/libmavconn/catkin_generated/mavlink.context.py"
  "-o"
  "/home/u23/WorkSpace/04.fixed_wing_sim/src/mavros/libmavconn/include/mavconn/mavlink_dialect.h"
  "/home/u23/WorkSpace/04.fixed_wing_sim/src/mavros/libmavconn/include/mavconn/mavlink_dialect.h.em")
  returned error code 1
Call Stack (most recent call first):
  /opt/ros/jade/share/catkin/cmake/em_expand.cmake:25 (safe_execute_process)
  mavros/libmavconn/CMakeLists.txt:57 (em_expand)
`

I checked the related files mentioned above, finding that it failed to generate the mavros/libmavconn/include/mavconn/mavlink_dialect.h file, which is empty. Is there any files or package I have missed or anything I need to do?
Please give me some instructions. 
Thanks.

Mavros: 0.18.6
ROS: Jade
Ubuntu: 14.04

**Comment by vooon** [comment]
https://github.com/mavlink/mavros/tree/master/mavros#source-installation

**Comment by yajingwang94** [comment]
Thanks a lot. I followed the instruction in this page, but always failed at the step: `rosdep install --from-paths src --ignore-src -y`
It reports :
![screenshot from 2017-02-21 09_44_51](https://cloud.githubusercontent.com/assets/18362945/23147575/de3ccce0-f81a-11e6-862a-ad73191ba7d4.png)
I returened to make my package again, and the error turns to be:
![screenshot from 2017-02-21 09_25_38](https://cloud.githubusercontent.com/assets/18362945/23147591/121618a0-f81b-11e6-8821-b3975f362d60.png)

Dose this mean that I failed to install the mavros package? I noticed that the `mavros_msgs `package should be included by` mavros` package. Is there anything I have missed?
